modules/ossm-migrating-workloads-using-the-istio-injection-label.adoc

```diff
@@ -0,0 +1,109 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/checklists/ossm-migrating-cluster-wide-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-workloads-using-the-istio-injection-label_{context}"]
+= Migrating workloads using the istio injection label
+
+Now you can migrate your workloads from the {SMProduct} 2.6 control plane to the {SMproduct} 3.0 control plane.
+
+[NOTE]
+====
+You can migrate workloads and gateways separately, and in any order. For more information, see "Migrating gateways".
+====
+
+.Procedure
+
+. Find the current `IstioRevision` for your {SMProduct} 3.0 control plane by running the following command:
```

Comment:
```suggestion
. Find the current `IstioRevision` resource for your {SMProduct} 3.0 control plane by running the following command:
```

- Object name is missing

https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#object-references

> An object reference is when you are referring to the actual instance of an API object, where the object name is important.